models/desired_lrp_test.go

```diff
@@ -1343,11 +1350,16 @@ var _ = Describe("DesiredLRPUpdate", func() {
 	var desiredLRPUpdate models.DesiredLRPUpdate
 
 	BeforeEach(func() {
-		desiredLRPUpdate.SetInstances(2)
-		desiredLRPUpdate.Routes = &models.Routes{
+		instances := int32(2)
+		desiredLRPUpdate.Instances = &instances
+		routes := &models.Routes{
 			"foo": &json.RawMessage{'"', 'b', 'a', 'r', '"'},
 		}
-		desiredLRPUpdate.SetAnnotation("some-text")
+
+		desiredLRPUpdate.Routes = routes
+		annotation := "some-text"
+		desiredLRPUpdate.SetAnnotation(&annotation)
+		// desiredLRPUpdate.Annotation = &annotation
```

Comment:
Delete?

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)